src/Entity/Talk.php

```diff
@@ -46,9 +46,15 @@ class Talk
      */
     private $submits;
 
+    /**
+     * @ORM\OneToMany(targetEntity=User::class, mappedBy="talks")
+     */
+    private $authors;
```

Comment:
Hmm, I think there is a design issue here. Actually the data schema is:
User 1->n Submit n->1 Talk
Here, you are adding a direct relation between User and Talk, but this relation exists, indirectly through Submit. 
So, you might be able to retrieve all talks of a user through there submits, with something like:

```
    public function getAuthors(): string
    {
        return array_reduce($this->getSubmits(), function ($r, Submit $submit) {
            foreach ($submit->getUsers() as $user) {
                $r = '' === $r ? $user->getName() : $r . ', ' . $user->getName();
            }

            return $r;
        }, '');
    }
```

(I did not test this code)

Reply:
Oh sure, it looks a lot better indeed and adresses what is said above.
Thanks !

Reply:
And BTW, the code here is ugly. Use a template in easy admin for that